detector/detector.go

```diff
@@ -145,6 +150,7 @@ func startNpdHttpServer(context *cli.Context) error {
 	})
 	http.HandleFunc("/v1/health/", healthCheckHandler)
 	http.HandleFunc("/v1/nodehealth/", nodeHealthHandler)
+	http.Handle("/metrics", metricsHandler(reg))
```

Comment:
I don't know if this should be behind `/v1/metrics` or not. It's unlikely to change based on API version, but if we prefer, for consistency, this can be changed.

Reply:
Thanks @fcuny-rbx! Yeah let's keep it consistent and make the endpoint `/v1/metrics/` (With a `/` at the end).